src/frontend/app.ts

```diff
@@ -230,19 +231,19 @@ export default (octokitApp: App): Express => {
 	app.get(
 		"/jira/configuration",
 		csrfProtection,
-		verifyJiraMiddleware,
+		verifyJiraMiddleware(TokenType.normal),
```

Comment:
How do we know which ones are normal and which ones are not?

Reply:
So endpoints which are called from Jira they contain a proper "qsh" parameter. So for those endpoints we use Normal tokens. 

Tokens which we generate inside iFrame they are so-called "Context" tokens and they don't have "qsh" set to a constant value "context-qsh". It is not empty to prevent this token being used as a universal authentication token and accessing any endpoints.

So for endpoints which are accessible from the iFrame we use context token type.

Reply:
In that case, I'd recommend you use 2 different functions for the middleware and not having to input the token type to a function.